packages/listener-solidarity/src/components/Msrs/index.ts

```diff
@@ -0,0 +1,68 @@
+import { User } from "../../types";
+import { getRaceColor } from "../../utils";
+import logger from "../../logger";
+import axios, { AxiosError } from "axios";
+
+type CreateMsrResponse = {
+  data: {
+    msrId: number;
+  };
+};
+
+const log = logger.child({ labels: { process: "createMsrs" } });
+
+function getMsrPayload(msr: User) {
+  return {
+    msrZendeskUserId: (msr.user_id as unknown) as bigint,
+    email: msr.email,
+    phone: msr.user_fields.whatsapp,
+    firstName: msr.name,
+    city: msr.user_fields.city,
+    state: msr.user_fields.state,
+    neighborhood: msr.user_fields.neighborhood,
+    zipcode: msr.user_fields.cep ? msr.user_fields.cep : "not_found",
```

Comment:
remover caracteres especiais

Reply:
esse é campo aberto? se for, acho que é uma boa deixar só números tbm